make/docker.mk

```
@@ -28,6 +28,7 @@ BUILDX_TARGET_PLATFORMS := $(subst $(space),$(comma),$(IMAGE_TARGET_PLATFORMS))
 DOCKER_BUILD_IMAGES := \
 	controller:$(PROJECT_DIR)/Dockerfile:$(PROJECT_DIR) \
 	quick-start:$(PROJECT_DIR)/install/quick-start/Dockerfile:$(PROJECT_DIR) \
+	setup-opensearch-cluster:$(PROJECT_DIR)/install/plane-setup/observability-plane/Dockerfile:$(PROJECT_DIR) \
```

Looks like the image name is different here
```suggestion
	opensearch-cluster-setup:$(PROJECT_DIR)/install/plane-setup/observability-plane/Dockerfile:$(PROJECT_DIR) \
```

---

Updated the name through https://github.com/openchoreo/openchoreo/pull/788/commits/e1dd4197c10250e2f714bd17e8679479830be496